kuma/revision/test_views.py

```diff
@@ -0,0 +1,52 @@
+import os
+from urlparse import urljoin
+
+import pytest
+import requests_mock
+
+from kuma.core.urlresolvers import reverse
+from kuma.wiki.constants import KUMASCRIPT_BASE_URL
+
+
+@pytest.mark.parametrize('method', ['get', 'head'])
+def test_revision_hash(client, db, method):
```

Comment:
Huh. It is sad that this requires a database. Not sure if I should blame Django or our middleware.

Reply:
I felt the same. I had originally left the `db` fixture off thinking it didn't need it, and was surprised when it did, not realizing that our middleware forced the issue.